main.py

```
@@ -80,8 +80,11 @@ def setup_logging(level: str = "INFO"):
     if enable_xiaozhi:
         logger.info("Starting Synology MCP Server with Xiaozhi Bridge")
         logger.info("Supports BOTH Xiaozhi and Claude/Cursor simultaneously")
+    elif config.http_enabled:
+        logger.info(f"Starting Synology MCP Server over Streamable HTTP")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)